hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/cache/CacheAddPartitionLostListenerMessageTask.java

```diff
@@ -61,9 +61,8 @@ public void partitionLost(CachePartitionLostEvent event) {
                 new InternalCachePartitionLostListenerAdapter(listener);
         final EventFilter filter = new CachePartitionLostEventFilter();
         final CacheService service = getService(CacheService.SERVICE_NAME);
-        final EventRegistration registration = service.getNodeEngine().
-                getEventService().registerListener(AbstractCacheService.SERVICE_NAME,
-                parameters.name, filter, listenerAdapter);
+        final EventRegistration registration = service.getNodeEngine().getEventService()
+                    .registerListener(ICacheService.SERVICE_NAME, parameters.name, filter, listenerAdapter);
         final String registrationId = registration.getId();
         endpoint.setListenerRegistration(CacheService.SERVICE_NAME, parameters.name, registrationId);
```

Comment:
Good catch. I have updated most of them as ICacheService.SERVICE_NAME but I had missed a few of them :)